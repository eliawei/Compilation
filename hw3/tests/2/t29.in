void c(byte foo) {
    print("c");
}

void foo(bool c) {
    return;
}

void main() {
    return;
}