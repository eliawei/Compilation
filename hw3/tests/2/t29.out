---end scope---
foo BYTE -1
line 5: identifier c is already defined
